src/mgmt/tests/srv_drpc_tests.c

```diff
@@ -786,13 +786,205 @@ test_drpc_list_pools_success_with_pools(void **state)
 	D_FREE(resp.body.data);
 }
 
+/*
+ * dRPC List Containers setup/teardown
+ */
+
+static int
+drpc_list_cont_setup(void **state)
+{
+	mock_ds_mgmt_pool_list_cont_setup();
+
+	return 0;
+}
+
+static int
+drpc_list_cont_teardown(void **state)
+{
+	mock_ds_mgmt_pool_list_cont_teardown();
+
+	return 0;
+}
+
+/*
+ * dRPC List Containers tests
+ */
+static void
+pack_list_cont_req(Drpc__Call *call, Mgmt__ListContReq *req)
+{
+	size_t	len;
+	uint8_t	*body;
+
+	len = mgmt__list_cont_req__get_packed_size(req);
+	D_ALLOC(body, len);
+	assert_non_null(body);
+
+	mgmt__list_cont_req__pack(req, body);
+
+	call->body.data = body;
+	call->body.len = len;
+}
+
+static void
+setup_list_cont_drpc_call(Drpc__Call *call, char *uuid)
+{
+	Mgmt__ListContReq lc_req = MGMT__LIST_CONT_REQ__INIT;
+
+	lc_req.uuid = uuid;	/* pool UUID */
```

Comment:
Suggestion - Maybe the input param should be pool_uuid so the comment would be unnecessary

Reply:
I see. My thinking in the pool.proto protobuf definition was that because ListContainers is an operation on a pool that uuid would be understood to be a pool UUID. I'd like to keep the naming. Could remove this comment if it's necessary, although I don't see it is a significant problem.

Reply:
Had to merge master and make changes, so removed the comment.